lib/peep.rb

```diff
@@ -0,0 +1,27 @@
+require 'pg'
+
+class Peep
+
+  def self.create(peep:, name:, username:)
+
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    connection.exec("INSERT INTO peeps (peep, name, username) VALUES('#{peep}','#{name}','#{username}');")
+  end
```

Comment:
You may have not made it that  far on the bookmark challenges, but connection.exec_params is best best practice compared to just connection.exec. Really interesting thing to look into if you get a chance, it basically helps stop your app being altered by SQL injections. (it's covered on step 12 of the bookmark challenge)

Reply:
Yeah I was out most of that week with tech issues, thanks I'll check it out!